code/game/objects/items.dm

```diff
@@ -1066,3 +1066,25 @@ var/global/list/image/blood_overlays = list()
 
 /obj/item/animationBolt(var/mob/firer)
 	new /mob/living/simple_animal/hostile/mimic/copy(loc, src, firer, duration=SPELL_ANIMATION_TTL)
+
+/obj/item/proc/is_worn(mob/user)
+	var/mob/living/carbon/monkey/M
+	var/mob/living/carbon/human/H
+	if(istype(user, /mob/living/carbon/monkey))
+		M = user
+	if(istype(user, /mob/living/carbon/human))
+		H = user
+	if(!(H || M))
+		return FALSE
+	if(H)
+		for(var/bit = 0 to 15)
+			bit = 1 << bit
+			if(bit & slot_flags)
+				if(H.get_item_by_flag(bit) == src)
+					return TRUE
+	else
```

Comment:
It calls the highest available defined version of that proc it can.

Meaning the latter.